Cargo.toml

```diff
@@ -27,7 +27,7 @@ tokio-core = "0.1"
 tokio-service = "0.1"
 tokio-uds = "0.1"
 # Temporary until zero-sized-chunk-problem lands in crates.io.
```

Comment:
What about this comment? Is it still actual?

Reply:
yes, I think it fixed in 0.11.2